plugins/database/mysql/mysql.go

```diff
@@ -262,9 +210,9 @@ func (m *MySQL) RotateRootCredentials(ctx context.Context, statements []string)
 		return nil, errors.New("username and password are required to rotate")
 	}
 
-	rotateStatents := statements
-	if len(rotateStatents) == 0 {
-		rotateStatents = []string{defaultMySQLRotateRootCredentialsSQL}
+	rotateStatements := statements
+	if len(rotateStatements) == 0 {
+		rotateStatements = []string{defaultMySQLRotateRootCredentialsSQL}
```

Comment:
Do we want to set a default rotation statement below in SetCredentials too? If not we at least should check if the statements is empty and not call `executePreparedStatmentsWithMap`